cli/command/secret/create.go

```diff
@@ -25,7 +25,7 @@ func newSecretCreateCommand(dockerCli *command.DockerCli) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "create [name]",
+		Use:   "create [OPTIONS] SECRET [SECRET...]",
```

Comment:
@ehazlett 
Does secret create support multiple secrets? I think  it only need support a secret as other create.
If it should be, I will open a PR for it.
Thanks